docs/docs/50-configuration/06-encryption-providers.md

```diff
@@ -0,0 +1,102 @@
+# Encryption Providers
+
+## Setting up Google Cloud KMS Encryption Provider
+
+## Setting up Azure Key Vault Encryption Provider
+
+This guide explains how to set up an Azure VM with a managed identity that can access an Azure Key Vault for encryption operations.
+
+### Prerequisites
+
+- Azure CLI installed and logged in (`az login`)
+- A subscription with permissions to create resources
+
+### 1. Create Resource Group
+
+```bash
+# Create a resource group
+az group create \
+  --name your-resource-group \
+  --location eastus
+```
+
+### 2. Create Key Vault and Key
+
+```bash
+# Create a Key Vault with RBAC authorization
+az keyvault create \
+  --name your-keyvault-name \
+  --resource-group your-resource-group \
+  --location eastus \
+  --enable-rbac-authorization true
+
+# Create an RSA key in the Key Vault
+az keyvault key create \
+  --name your-key-name \
+  --vault-name your-keyvault-name \
+  --size 2048 \
+  --kty RSA
```

Comment:
RSA is generally bad.. We should recommend something like `EC`

Reply:
(We can change `--kty` to `EC` and remove the `--size` parameter and that will work.

Reply:
I retract this statement.. looks like EC doesnt work for this.